source/includes/table-permissions-self-hosted.rst

```diff
@@ -4,47 +4,58 @@
 
 .. list-table::
    :header-rows: 1
+   :stub-columns: 1
```

Comment:
**Question**

Just curious what exactly is a stub-column?

Reply:
`:stub-columns:` are the counterpart to `:header-rows:`.  It applies styling to columns bolding the cells from left to right.  Usually, it's one or the other but here I think we need to emphasize both.